source/backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -77,6 +77,10 @@ private static IQueryable<Entity.PimsProperty> GenerateCommonPropertyQuery(this
             {
                 query = query.Where(p => EF.Functions.Like(p.Address.StreetAddress1, $"%{filter.Address}%") || EF.Functions.Like(p.Address.MunicipalityName, $"%{filter.Address}%"));
             }
+            if (!string.IsNullOrWhiteSpace(filter.PlanNumber))
```

Comment:
same here, plan number should be an exact match unlike pid/pin. This is because plan numbers do not have a fixed length so someone looking for plan VIP8 will find thousands of matches in the province for plans VIP8X, VIP8XX, and VIP8XXX etc.